### PR TITLE
Added Unit Tests For Job Service

### PR DESCRIPTION
**Pull request by code-incubus**
Signed-off-by: code-incubus nemanja.milutinovic@comtrade.com

Added unit test scenarios for Job Service.

Related Issue
None

Test scenarios that are added in this PR are checking functionalities regarding Jobs. All scenarios are added to JobService.feature. Scenarios are testing if Jobs are correctly created and edited.

Screenshots
/

Any side note on the changes made
/

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
Hi @code-incubus,
I have checked this PR and it looks OK. There is only one thing - please rebase it to the latest develop, because you are missing quite a lot commits. 
After that It will be approved and merged (I can see that there was no problem with the tests).

**Comment by code-incubus** [comment]
@LeoNerdoG Done.

**Comment by LeoNerdoG** [comment]
Hi @code-incubus,
I have checked this and now everything is OK. 
@Coduz, can you please merge this after the tests complete?

**Review by LeoNerdoG — APPROVED** [review]

**Review by Coduz — APPROVED** [review]

**Pull request merged**
